### PR TITLE
Add serverPath option

### PR DESCRIPTION
**Pull request by domoritz**
Follow up from #12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -306,10 +306,14 @@ async function wasm_pack(cx, state, dir, source, id, options) {
                     ${prelude}
 
                     export default async (opt = {}) => {
-                        let {importHook} = opt;
+                        let {importHook, serverPath} = opt;
 
                         let path = ${import_wasm};
 
+                        if (serverPath != null) {
+                            path = serverPath + path.split('/').pop();
```
Comment (Pauan) [inline index.js]:
```suggestion
                            path = serverPath + /[^\/\\]*$/.exec(path)[0];
```

I think we should handle `\` as well, for extra robustness.

**Review by Pauan — CHANGES_REQUESTED** [review]
Overall this looks good, just one minor change.

**Comment by domoritz** [comment]
Great. I added your suggestion.

**Comment by Pauan** [comment]
Thanks!

**Pull request merged**